studio/src/windowManagement/contentWindows/ContentWindowBuildView/ContentWindowBuildView.js

```diff
@@ -82,15 +82,15 @@ export class ContentWindowBuildView extends ContentWindow {
 			hasDownArrow: true,
 			colorizerFilterManager,
 			onClick: () => {
-				const studio = getStudioInstance();
-				const projectSettings = studio.projectManager.projectSettings;
-				const assetManager = studio.projectManager.assetManager;
-				if (!projectSettings || !assetManager) return;
-				const popover = studio.popoverManager.createPopover();
-				// eslint-disable-next-line no-new
-				new EntryPointManager(popover, projectSettings, assetManager, this.persistentData);
-
-				popover.setNeedsCurtain(false);
+				const projectSettings = this.studioInstance.projectManager.projectSettings;
```

Comment:
Nice cleanup! There's a few more calls to `getStudioInstance()` in this file. Feel free to replace those as well, then the `getStudioInstance` import can be completely removed.